src/main/java/frc/robot/subsystems/Storage.java

```diff
@@ -6,31 +6,43 @@
 public class Storage extends SubsystemBase {
 
     public static final DigitalInput ballsInSwitch = new DigitalInput(StorageConstants.BALLS_IN_SWITCH_ID);
+    public static final DigitalInput ballsOutSwitch = new DigitalInput(StorageConstants.BALLS_OUT_SWITCH_ID);
     public static final WPI_VictorSPX polycord = new WPI_VictorSPX(StorageCosntants.POLYCORD_MOTOR_ID);
 
     public Storage() {
 
     }
 
     public void runPolycordForward(){
-        intakeFlywheels.set(ControlMode.PercentOutput, .2);
+        polycord.set(ControlMode.PercentOutput, .2);
     }
 
     public void runPolycordBackward(){
-        intakeFlywheels.set(ControlMode.PercentOutput, -.2);
+        polycord.set(ControlMode.PercentOutput, -.2);
     }
 
     public void stopPolycord(){
-        intakeFlywheels.set(ControlMode.PercentOutput, 0);
+        polycord.set(ControlMode.PercentOutput, 0);
     }
 
-    private boolean loadingIncomingBalls(){
-        if (!ballsInSwitch.get()) {
-            runPolycordForward();
-            return true;
+    /**If the system finished and pulled in the ball return false. While still running return true
+     * Runs when drive team presses a button.**/
+    public boolean loadIncomingBalls(){
```

Comment:
logic like this is for the command, the subsystem should only be base functionally